### PR TITLE
change the background color to white when selected

### PR DESCRIPTION
**Pull request by rrryy39**
selected

### CODE REVIEW

**Comment by moorepants** [comment]
You're first pull request! Nice!

**Comment by moorepants** [comment]
I just ran this on my computer and see one issue. If I click cells in the first column of any table the text does not change to black with the white background:
image

Either make those cells not-clickable or make them behave like your cells.

Make your changes, commit again, and push the branch. That will update this PR with new commits.

**Comment by rrryy39** [comment]
I changed the background color to gray and text color to black, if you do not like the theme, i can try to make the cell unclickable 
image

**Comment by moorepants** [comment]
I don't think that looks good. I liked what you had first, but you need to have all columns behave the same.

**Comment by rrryy39** [comment]
Now the first column's text is black when the cell is selected
image

**Comment by moorepants** [comment]
Good. This works for me.

File: bicycleparameters/cycle_app.py [hunk 1]
```diff
@@ -128,8 +128,10 @@
                                                      },
                                                      style_data_conditional=[
                                                          {
-                                                             'if': {'column_editable': False},
-                                                             'cursor': 'not-allowed'
+                                                            "if": {"state": "selected"},              # 'active' | 'selected'
```
Comment (moorepants) [inline bicycleparameters/cycle_app.py:131]: Don't leave commented old code like this. It is in the git history, so you can always get it back.

**Comment by moorepants** [comment]
Remove the old commented code and I'll merge this.

**Comment by rrryy39** [comment]
comments removed

**Comment by moorepants** [comment]
Great your first PR is in! On to the next :)

**Pull request merged**